apis/filesystem/src/test/java/org/jclouds/filesystem/FilesystemBlobStoreTest.java

```diff
@@ -491,6 +495,7 @@ public void testPutDirectoryBlobs() {
         assertTrue(blobStore.blobExists(CONTAINER_NAME, childKey));
 
         blobStore.removeBlob(CONTAINER_NAME, parentKey);
+        Uninterruptibles.sleepUninterruptibly(1, TimeUnit.SECONDS);
```

Comment:
Could we change `removeBlob` to loop until the blob is deleted to give user-facing consistency instead of just test consistency?